tests/schemas/valid/test_text_field_typeahead.json

```diff
@@ -0,0 +1,65 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.3",
+  "survey_id": "001",
+  "title": "Other input fields",
+  "theme": "default",
+  "description": "A questionnaire to test textfields",
+  "messages": {
+    "NUMBER_TOO_LARGE": "Number is too large",
+    "NUMBER_TOO_SMALL": "Number cannot be less than zero",
+    "INVALID_NUMBER": "Please enter an integer"
+  },
+  "metadata": [
+    {
+      "name": "user_id",
+      "type": "string"
+    },
+    {
+      "name": "period_id",
+      "type": "string"
+    },
+    {
+      "name": "ru_name",
+      "type": "string"
+    }
+  ],
+  "sections": [
+    {
+      "id": "default-section",
+      "groups": [
+        {
+          "blocks": [
+            {
+              "type": "Question",
+              "id": "country-block",
+              "question": {
+                "answers": [
+                  {
+                    "id": "country-answer",
+                    "label": "What is your country of birth?",
+                    "mandatory": false,
+                    "url": "https://gist.githubusercontent.com/rmccar/c123023fa6bd1b137d7f960c3ffa1fed/raw/368a3ea741f72c62c735c319ff7e33e3c1bfdc53/country-of-birth.json",
```

Comment:
I think we should make this URL valid and not commit a personal gist. Maybe just `"https://eq.ons.gov.uk`

Reply:
It is a valid url and doesn't matter what the url is? It's not being visited as part of tests.

Reply:
It's so that we don't need to commit someone personal gist...

Reply:
It doesn't matter? It can be any url.

Reply:
The point isn't that it can't be any url, the point is that I don't think it should link to an external gist. May as well make it https://abc.com if it doesn't matter. But okay I fold.